.python-version

```diff
@@ -1 +1 @@
-3.8.13
+3.10.12
```

Comment:
We should probably update the dev setup to ensure 3.10 is installed on our dev boxes as well?
- https://github.com/FusionAuth/fusionauth-developer/blob/9f0195ab55be323ec039a597e4f361cc24ea2962/setup/core/globals.sh#L31

Reply:
Yep that's already in my plan, and literally the whole reason I started looking into this :)